packages/client-twitter/src/post.ts

```diff
@@ -44,20 +44,25 @@ export const twitterActionTemplate =
 {{postDirections}}
 
 Guidelines:
-- Highly selective engagement
-- Direct mentions are priority
-- Skip: low-effort content, off-topic, repetitive
+- ONLY engage with content that DIRECTLY relates to character's core interests
+- Direct mentions are priority IF they are on-topic
+- Skip ALL content that is:
+  - Off-topic or tangentially related
+  - From high-profile accounts unless explicitly relevant
+  - Generic/viral content without specific relevance
+  - Political/controversial unless central to character
+  - Promotional/marketing unless directly relevant
```

Comment:
Thanks for doing this. This seems like a pretty meaningful change that will change how the bot interacts for everyone can we make this configurable.

Reply:
What I mean by that is adding a responseThreshold enum or something and have different meta prompting based off of the configured value? You could start with a low, high, medium

Reply:
all prompts should be overwrite-able in the character prompt key, this is just a change to the default. Your request would be better but seems out of scope.